posts/2024/2024_06_14_Inigo_week_3.rst

```diff
@@ -0,0 +1,35 @@
+Third Week into GSoC 2024: Replicating training parameters, approaching replication
+===================================================================================
+
+.. post:: June 14 2024
+    :author: Iñigo Tellaetxe
+    :tags: google
+    :category: gsoc
+
+
+What I did this week
+~~~~~~~~~~~~~~~~~~~~
```

Comment:
LGTM, apart from that if you could leave a blank line after the titles, like here, it would be better for reading.